.github/workflows/staging-odfe-cli.yml

```diff
@@ -0,0 +1,53 @@
+name: Staging ODFE CLI
+
+on:
+#  schedule:
+#    - cron: '0 12 * * *'
+  repository_dispatch:
+    types: [staging-odfe-cli]
+
+jobs:
+  Test-ODFE-CLI:
+    runs-on: ubuntu-18.04
+    name: Test-ODFE-CLI
+    strategy:
+      matrix:
+        go-version: [1.14]
+    steps:
+      - name: Set up AWS Cred
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ matrix.go-version }}
+      - uses: actions/checkout@v1
+      - name: Retrieve plugin tags
+        run: |
+          echo "tag_v=$(./bin/plugins-info client cutversion --require-install-false | tr ' ' '\n' | uniq | sort -V | head -n 1)" >> $GITHUB_ENV 
```

Comment:
Same issues

Reply:
Moved to my old code to directly pick the cut version as all the clients require install needs to be false.